src/eth1spec/base_types.py

```diff
@@ -240,9 +240,305 @@ def to_be_bytes(self) -> "Bytes":
         return self.to_bytes(byte_length, "big")
 
 
+class U256(int):
+    """
+    Unsigned positive integer, which can represent `0` to `2 ** 256 - 1`,
+    inclusive.
+    """
+
+    MAX_VALUE: "U256"
+
+    __slots__ = ()
+
+    @classmethod
+    def from_be_bytes(cls: Type, buffer: "Bytes") -> "U256":
```

Comment:
Can you really not return the class a method is a member of without double quoting it?

Reply:
Not sure! I think with the `from __future__ import annotations` line you can avoid it.